controller/persistence/service_policy_store.go

```diff
@@ -92,6 +92,13 @@ func (entity *ServicePolicy) GetEntityType() string {
 	return EntityTypeServicePolicies
 }
 
+func (entity *ServicePolicy) GetPolicyTypeName() string {
+	if entity.PolicyType == PolicyTypeBind {
+		return PolicyTypeBindName
+	}
+	return PolicyTypeDialName
+}
+
 type ServicePolicyStore interface {
 	NameIndexedStore
 	LoadOneById(tx *bbolt.Tx, id string) (*ServicePolicy, error)
```

Comment:
I didn't see any API tests

Reply:
Yes, I weighed writing test vs getting back to HA features and skipped them for now. I think we want them in the medium term, but in the short term, since this is advisor code, I thought it was okay.